.github/workflows/reusable-test.yml

```diff
@@ -35,6 +35,11 @@ jobs:
           node-version-file: ".nvmrc"
           cache: "yarn"
 
+      - name: 🦕 Setup deno
```

Comment:
I called it that because the previous step is called [⎔ Setup node](https://github.com/remix-run/remix/pull/4715/files#diff-e386584fe70c301924ab6a8d4656b54ef92ed1961e6eb6f77c155be3475615f6L32)